### PR TITLE
Misc Staff Area Fixes and Changes

### PR DESCRIPTION
**Pull request by ghickman**
This is a grab bag of changes to the staff area:
* Fix the project detail page to show its workspace list
* Make projects editable
* Add orgs (detail, list, edit) to the staff area
* Update some links to point to pages in the staff area
* Allow members to be added to/removed from orgs

### CODE REVIEW

**Review by CarolineMorton — APPROVED** [review]
Looks good to me. 

Thanks for clarifying re: `method_decorator` on slack. 

Nice to see such a comprehensive set of tests!!

**Comment by ghickman** [comment]
> can a user be a member of more than one organisation

Yes

> This might be something that a co-pilot might want

If they have access to the staff area, they have acess to every page in it!

File: tests/unit/staff/views/test_orgs.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+def test_orglist_success(rf, core_developer):
+    OrgFactory.create_batch(5)
+
+    request = rf.get("/")
+    request.user = core_developer
+
+    response = OrgList.as_view()(request)
+
+    assert response.status_code == 200
+    assert len(response.context_data["org_list"])
```
Comment (inglesp) [inline tests/unit/staff/views/test_orgs.py:142]: Missing `== 5`?

File: staff/views/orgs.py [hunk 2]
```diff
@@ -1,22 +1,57 @@
...
     template_name = "staff/org_detail.html"
 
+    def dispatch(self, request, *args, **kwargs):
+        self.object = get_object_or_404(Org, slug=self.kwargs["slug"])
+
+        return super().dispatch(request, *args, **kwargs)
+
+    def form_valid(self, form):
+        for user in form.cleaned_data["users"]:
+            self.object.memberships.create(user_id=user, created_by=self.request.user)
```
Comment (inglesp) [inline staff/views/orgs.py:26]: Could we end up with multiple memberships for a single user here?

File: tests/unit/staff/views/test_orgs.py (hunk 1 above)
Comment (ghickman) [reply tests/unit/staff/views/test_orgs.py:142]: oops, yes!

File: staff/views/orgs.py (hunk 2 above)
Comment (ghickman) [reply staff/views/orgs.py:26]: No, they're unique on Org & User so we should get an error then, but I'll also trim the available orgs down so we only see ones without a membership 👍

**Pull request merged**